### PR TITLE
docs: update zh-cn customisation guide

### PR DESCRIPTION
**Pull request by MaikoTan**
> [!Warning]
> This is the following up for #6047 , also fix several typos in English version.

^ GitHub's new feature, funny!

According to [Microsoft's localisation guide](https://learn.microsoft.com/en-us/globalization/localization/ministyleguides/mini-style-guide-sim-chinese) and [vinta's 盤古之白 convince](https://github.com/vinta/pangu.js), this PR do the following:

- Add missing lines in the previous PR.
- Replace all polite `您` to a normal `你`.
- Insert spaces between Chinese characters and western characters or symbols (the `Pangu`-ism).
- Captalize all English words, unless it has a special spell rule.

------

@Echoring @ShadyWhite

### CODE REVIEW

File: docs/zh-CN/CactbotCustomization.md [hunk 1]
```diff
@@ -237,33 +238,33 @@ Options.Triggers.push([
...
+我们推荐阅读 [触发器指南](RaidbossGuide.md) 以了解如何撰写 Cactbot 的触发器，当然你也可以直接看 [ui/raidboss/data](../../ui/raidboss/data) 中现有的触发器代码。
 
-## Raidboss时间轴自定义
+## Raidboss 时间轴自定义
 
-一些自定义操作可以通过 [cactbot 配置界面](#使用cactbot配置界面) 实现。你可以在这个界面隐藏或重命名现有的时间轴条目，也可以添加自定义时间轴条目等。
+一些自定义操作可以通过 [Cactbot 配置界面](#使用-cactbot-配置界面) 实现。你可以在这个界面隐藏或重命名现有的时间轴条目，也可以添加自定义时间轴条目等。
 
-仅当你需要做的操作超出了配置界面所能提供的范围时才考虑使用本章节的操作，比如完全替换整个时间轴。替换时间轴的操作与 [替换触发器](#替换raidboss触发器) 类似。
+仅当你需要做的操作超出了配置界面所能提供的范围时才考虑使用本章节的操作，比如完全替换整个时间轴。替换时间轴的操作与 [替换触发器](#替换-raidboss-触发器) 类似。
```
Comment (Souma-Sumire) [inline docs/zh-CN/CactbotCustomization.md:255]: 这个“替换raidboss触发器”链接一直是无效的，如果能趁着这次机会改了会是极好的。

Comment (MaikoTan) [reply docs/zh-CN/CactbotCustomization.md:255]: I think now it is utilisable, isn't it?

**Comment by quisquous** [comment]
Hi, sorry for the hassle. I've moved cactbot to OverlayPlugin/cactbot (and probably there are some minor url conflicts that need to be resolved as well). You should move this PR over there to <https://github.com/OverlayPlugin/cactbot/> instead of here to get landed.

**Comment by MaikoTan** [comment]
Sure, I'll instead open a PR there.

**Pull request closed without merging**